### PR TITLE
Creating Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by Jawad-Nazari**
### In this PR , I did the following: 

- Created Person class
- Created  Student class inherited from Person class
- Created Teacher class inherited from Person class
- Cleared Linter errors

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Person
+  attr_accessor :id, :name, :age
```
Comment (prolajumokeoni) [inline person.rb]: - [ ] Kindly set your id `attr_reader :id`,  id should only be able to be retrieved so only the Getter is defined

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Jawad-Nazari 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The classes to represent students, person and teachers have been implemented💪🏿



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (Jawad-Nazari) [reply person.rb]: Hi @prolajumokeoni ,

Thanks for your time and reviewing my code. I will apply the required changes.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Jawad-Nazari 

Your project is complete! Great job  🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- classes well added
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**